infra/bin/infra.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as cdk from 'aws-cdk-lib';
+import 'source-map-support/register';
+import { AmplifyInfraStack } from '../lib/stack';
+
+const app = new cdk.App();
+const env: cdk.Environment = {
+  account: process.env.CDK_DEFAULT_ACCOUNT,
+  region: process.env.CDK_DEFAULT_REGION,
+};
+
+new AmplifyInfraStack(app, 'AmplifyInfraStack', {
```

Comment:
nit/question: could this be named something like "AmplifyFutterIntegrationTestResources" (dk char limits) as "InfraStack" doesn't really add any info to a cloudformation stack. Could this fill in the description field for this stack as well as other nested stacks?